controllers/search_test.go

```diff
@@ -118,6 +140,16 @@ func TestSearch_controller(t *testing.T) {
 		t.Fatalf("Failed to get configmap %s: %v", "search-indexer", err)
 	}
 
+	//check for configmap
+	configmap3 := &corev1.ConfigMap{}
+	err = cl.Get(context.TODO(), types.NamespacedName{
+		Name: "search-postgres",
+	}, configmap3)
+
+	if err != nil {
+		t.Fatalf("Failed to get configmap %s: %v", "search-postgres", err)
+	}
+
```

Comment:
Could we also verify the contents of the configMap?

Reply:
Updated the test to verify certain contents of the configMap.